RecoLocalMuon/GEMSegment/plugins/MuonSegFit.cc

```diff
@@ -332,7 +326,6 @@ MuonSegFit::SMatrix12by4 MuonSegFit::derivativeMatrix() {
   int row = 0;
   
   for( MuonRecHitContainer::const_iterator it = hits_.begin(); it != hits_.end(); ++it) {
-    if (row > 11) break; // temp due to max size of matrix
```

Comment:
shouldn't this be put back in with ``` row >= MaxHits2 ``` ?
and also maybe if row is at least MaxHits2 to write a LogError